bamt/utils/gmm_wrapper.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+from sklearn.mixture import GaussianMixture
+from sklearn.mixture._gaussian_mixture import _compute_precision_cholesky
+import warnings
+from scipy.stats import multivariate_normal
+
+
+class GMM:
+    def __init__(
+        self,
+        n_components=1,
+        priors=None,
+        means=None,
+        covariances=None,
+        random_state=None,
```

Comment:
Пример типизации по новому стандарту питона (с версии 3.10)
```suggestion
        n_components: int = 1,
        priors: np.ndarray | None = None,
        means: np.ndarray | None = None,
        covariances: np.ndarray | None = None,
        random_state: int | None = None,
```